doc/classes/Viewport.xml

```diff
@@ -95,14 +95,20 @@
 				Returns the visible rectangle in global screen coordinates.
 			</description>
 		</method>
+		<method name="gui_get_active_focus_layer" qualifiers="const">
+			<return type="int" />
+			<description>
+			</description>
+		</method>
 		<method name="gui_get_drag_data" qualifiers="const">
 			<return type="Variant" />
 			<description>
 				Returns the drag data from the GUI, that was previously returned by [method Control._get_drag_data].
 			</description>
 		</method>
-		<method name="gui_get_focus_owner">
+		<method name="gui_get_focus_owner" qualifiers="const">
 			<return type="Control" />
+			<argument index="0" name="arg0" type="int" />
```

Comment:
New argument needs to be mentioned in the description.
Also you should specify the name in `bind_method`.